mlflow/pipelines/cards/histogram_generator.py

```diff
@@ -0,0 +1,124 @@
+"""
+Generates facets_overview histogram message for numeric features.
+"""
+
+from facets_overview.feature_statistics_pb2 import Histogram
+
+
+def generate_equal_height_histogram(quantiles, num_buckets: int) -> Histogram:
+    """Generates the equal height histogram from the input quantiles. The quantiles are assumed to
+    be ordered and corresponding to equal distant percentiles.
+
+    :param quantiles: The quantiles that capture the frequency distribution.
+    :param num_buckets: The number of buckets in the generated equal height histogram.
+    :return: An equal height histogram or None if inputs are invalid.
+    """
+    if (
+        (len(quantiles) == 0)
+        or (len(quantiles) < 3)
+        or (len(quantiles) % 2 == 0)
```

Comment:
Why does the length of quantiles need to be even? E.g. is 3 buckets demarcated by 4 quantiles invalid?

Reply:
(I realize we're pulling this in from Databricks, but I'd still like to understand better if possible :D)

Reply:
Spoke to Shupeng Sun. It seems like we can remove them and ease the restrictions. 

We still want to keep `(len(quantiles) < 3)` since we ideally don't want to just show 1 bucket. So still keeping that condition.